src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -41,6 +44,13 @@
 	private transient String priorityParameterName;
 	private List<LockableResource> resources;
 
+
+	/**
+	 * Only used when this lockable resource is tried to be locked by {@link LockStep},
+	 * otherwise (freestyle builds) regular Jenkins queue is used.
+	 */
+	private List<QueuedContextStruct> queuedContexts = new ArrayList<QueuedContextStruct>();
```

Comment:
As far as my limited understanding goes, this field is serialized in the configuration file once `save()` is called. Since whenever something is added or removed to this list, `save()` is called, this should survive a restart.